src/tests/e2e.test.ts

```diff
@@ -0,0 +1,291 @@
+import { JsonRpcProvider } from '@ethersproject/providers'
+import { getTokenByChainAndSymbol } from '@indexcoop/tokenlists'
+import { expect } from 'chai'
```

Comment:
Locally, I've been having issues running the tests. Will have to investigate more why.